stk/calculators/optimization/macromodel.py

```diff
@@ -1042,6 +1110,96 @@ def __init__(self,
                          minimum_gradient=minimum_gradient,
                          use_cache=use_cache)
 
+    @staticmethod
+    def _check_params(
+        temperature,
+        conformers,
+        simulation_time,
+        time_step,
+        eq_time,
+        minimum_gradient,
+        maximum_iterations
+    ):
+        """
+        Check if the optimization parameters are valid for MacroModel.
+
+        MacroModel ``.com`` files have well-defined column widths and
+        special ways to implement too long input. This method raises
+        `MacroModelInputError` exception if the input cannot be
+        converted into a valid ``.com`` file entry.
```

Comment:
Same as above, I feel this paragraph is bordering on implementation detail and raising comment should be in a Raises section.